HW_1/task06.py

```diff
@@ -0,0 +1,21 @@
+# Задача 6: Вы пользуетесь общественным транспортом?
+# Вероятно, вы расплачивались за проезд и получали билет с номером.
+# Счастливым билетом называют такой билет с шестизначным номером, где сумма первых трех цифр равна сумме последних трех.
+# Т.е. билет с номером 385916 – счастливый, т.к. 3+8+5=9+1+6.
+# Вам требуется написать программу, которая проверяет счастливость билета.
+# Пример:
+# 385916 -> yes
+# 123456 -> no
+
+
+numTicket = input('Введите номер билета(6-тизначное число): ')
+resultOne = 0
+resultTwo = 0
+n = int(len(numTicket))
+for i in range(n+1):
+    if i < n / 2:
+        resultOne += int(numTicket[i])
+for i in range(n):
+    if i >= n / 2:
+        resultTwo += int(numTicket[i])
```

Comment:
Вообще, это можно объединить под одним циклом
```suggestion
for i in range(n):
    if i < n / 2:
        resultOne += int(numTicket[i])
    else:
        resultTwo += int(numTicket[i])
```